### ISSUE TITLE
Mobile horizontal padding discrepancy

### ISSUE THREAD

**Issue by zackkrida** [WordPress/openverse#2590]
## Description

Banners have less horizontal padding than the site contents on mobile. We should adjust one to match the other, or find a normalized value between the two.

Might want to consider maximizing our use of horizontal space and making the padding a bit smaller?

## Reproduction

1. Visit https://openverse.org in a private/incognito mode on a mobile device.

## Screenshots

<img width="450" alt="CleanShot 2023-07-07 at 14 05 38@2x" src="https://github.com/WordPress/openverse/assets/6351754/7195bd91-c62f-4389-8ed7-1bbc24c2d45a">

**Comment by zackkrida** [comment]
Personally, I think I like a horizontal padding of `3` (using our scale) on mobile:

<img width="402" alt="CleanShot 2023-07-07 at 15 38 55@2x" src="https://github.com/WordPress/openverse/assets/6351754/b7484826-4210-4cc4-99e4-f956fe5fa349">

**Comment by fcoveram** [comment]
I don't think the banner should match the spacings of search result page layout. This element, similar to the global player, was thought to match the screen area. Therefore, it will look different in the homepage, the search result, and in the future, the single result in modal.

This situation does not impact the search experience significantly to be tagged as priority medium, compared to #547 that has been resting there for months and involved a layout improvement of the image result page.

**Comment by zackkrida** [comment]
Agreed, it should be low. If you think the banner spacing looks good across device sizes please close this issue 👍
